docs/source/index.rst

```diff
@@ -3,8 +3,37 @@
 Composer
 ========
 
-.. include:: ../../README.md
-   :parser: myst_parser.sphinx_
+`Composer <https://github.com/mosaicml/composer>`_ is a library for training neural networks better, faster, and cheaper.
```

Comment:
I'm OK with a shorter portion, but can you use `start-after` and `end-before` to include the file rather than copy-and-pasting? See https://docutils.sourceforge.io/docs/ref/rst/directives.html#include.

(It was challenging trace down all of the references to the Resenet and GPT costs, so would like to have them in as few places as possible.)